From 9518493d9f84d9785bd5a60def1943cfc5ee0f57 Mon Sep 17 00:00:00 2001
From: Nitin Garg <nitin.garg@freescale.com>
Date: Thu, 25 Oct 2012 18:26:00 -0500
Subject: [PATCH 087/310] ENGR00231269: Fix the LiveWallpaper on iMX53
 Jellybean

Fix the LiveWallpaper issue on Jellybean, ar3k bluetooth
firmware path and few cleanups.

Change-Id: I843d454d003019c9dbdd34676f72a48666cf7305
Signed-off-by: Nitin Garg <nitin.garg@freescale.com>
---
 imx5x/imx5x.mk | 10 ++++------
 imx5x/init.rc  |  1 +
 2 files changed, 5 insertions(+), 6 deletions(-)

diff --git a/imx5x/imx5x.mk b/imx5x/imx5x.mk
index d6d9764..164eec0 100644
--- a/imx5x/imx5x.mk
+++ b/imx5x/imx5x.mk
@@ -14,16 +14,15 @@ PRODUCT_PACKAGES += \
 	LiveWallpapersPicker			\
 	MagicSmokeWallpapers			\
 	VisualizationWallpapers			\
-	CubeLiveWallpapers			\
+	Galaxy4					\
+	NoiseField				\
+	PhaseBeam				\
 	HoloSpiralWallpaper			\
 	Gallery2				\
 	SoundRecorder				\
-        Camera                                  \
-        LegacyCamera                            \
 	FSLOta					\
 	VideoEditor				\
 	PinyinIME				\
-	libRS					\
 	librs_jni				\
 	pppd					\
 	chat					\
@@ -243,7 +242,7 @@ PRODUCT_PACKAGES += libubi ubinize ubiformat ubiattach ubidetach ubiupdatevol ub
 PRODUCT_PACKAGES += \
     com.android.future.usb.accessory
 
-PRODUCT_AAPT_CONFIG := normal mdpi
+PRODUCT_AAPT_CONFIG := normal xlarge mdpi
 
 PRODUCT_COPY_FILES +=	\
 	device/fsl/common/input/Dell_Dell_USB_Keyboard.kl:system/usr/keylayout/Dell_Dell_USB_Keyboard.kl \
@@ -261,7 +260,6 @@ PRODUCT_COPY_FILES +=	\
 
 PRODUCT_TAGS += dalvik.gc.type-precise
 
-
 # for property
 PRODUCT_DEFAULT_PROPERTY_OVERRIDES := \
 	persist.sys.usb.config=mtp
diff --git a/imx5x/init.rc b/imx5x/init.rc
index 7d09d0c..42f21e4 100644
--- a/imx5x/init.rc
+++ b/imx5x/init.rc
@@ -47,6 +47,7 @@ loglevel 3
 
 # Backward compatibility
     symlink /system/etc /etc
+    symlink /system/lib /lib
     symlink /sys/kernel/debug /d
 
 # Right now vendor lives on the same filesystem as system,
-- 
1.8.0

